fedimint-testing/src/gateway.rs

```diff
@@ -115,10 +114,7 @@ impl GatewayTest {
             address.clone(),
             cli_password.clone(),
             None, // Use default Network which is "regtest"
-            RoutingFees {
-                base_msat: 0,
-                proportional_millionths: 0,
-            },
+            DEFAULT_FEES,
```

Comment:
Does this break any of the outgoing payment tests? I thought we had a test that assumed no fees were taken out. Either way, this is changing the fees so I'm confused why more of the tests didn't require changes

Reply:
One test is broken, because I didn't break it. Actually revising this so I don't change the default fees, but since my intent is to have better coverage for fees configured, I added ppm part to `test_can_change_routing_fees`